src/components/ContactForm/ContactForm.tsx

```diff
@@ -24,6 +25,12 @@ export default function ContactForm({
       try {
         // Handle submit actions
         // console.log('Handle submit', values)
+        await sendContactRequest({
```

Comment:
**nit:** This should be fired from this `action` that is on your previous PR. Firstly, contact request should be created, then if successfully created, emails should be sent

Reply:
Done.